config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -452,6 +452,31 @@ postsubmits:
       - name: k8s-gcr-prod-service-account-creds
         secret:
           secretName: k8s-gcr-prod-service-account
+  kubernetes/community:
+  - name: post-community-tempelis-apply
+    cluster: test-infra-trusted
+    decorate: true
+    branches:
+      - ^master$
+    run_if_changed: '^communication/slack-config'
+    spec:
+      containers:
+        - image: gcr.io/k8s-tempelis/tempelis:latest
```

Comment:
a) Can this follow our standard tagging format?
b) If you use a mutable tag, you'll need to set always pull

Reply:
Changed it to use `v20190412-7c9b6e51`.